<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home</class>
 <widget class="QWidget" name="home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>193</width>
     <height>247</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_aggProvedores">
      <property name="text">
       <string>AGREGAR PROVEDORES</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_usuarios">
      <property name="text">
       <string>REGISTRAR USUARIOS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_clientes">
      <property name="text">
       <string>REGISTRAR CLIENTES</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_registrarEmpleado">
      <property name="text">
       <string>REGISTRAR EMPLEADO</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_registrar">
      <property name="text">
       <string> REGISTRAR PRODUCT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_consultar">
      <property name="text">
       <string>CONSULTAR PRODUCTOS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_registrarVentas">
      <property name="text">
       <string>REGISTRAR VENTAS</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
